websites/P/Paramount+/presence.ts

```diff
@@ -0,0 +1,213 @@
+const presence = new Presence({
+  clientId: "821433038335377418"
+}), 
+  strings = presence.getStrings({
+    play: "presence.playback.playing",
+    pause: "presence.playback.paused",
+    live: "presence.activity.live",
+    search: "presence.activity.searching"
+  });
+
+  let title: string,
+    seasonEpi: string,
+    movTitle: string,
+    vidArea: Element,
+    vidMdTl: Element,
+    elapsed: number = undefined,
+    oldUrl: string = undefined;
+
+  presence.on("UpdateData", async () => {
+
+    vidArea = document.querySelector(".video__player-area");
+    let video: HTMLVideoElement = null;
+    const presenceData: PresenceData = { 
+      largeImageKey: "logo"
+    }, href = window.location.href,
+      path = window.location.pathname;
+
+    if (href !== oldUrl) {
+      oldUrl = href;
+      elapsed = Math.floor(Date.now() / 1000);
+    }
+
+    presenceData.state = undefined;
```

Comment:
No, undefined... ``delete presenceData.state`` but you don't need to set it to undefined before even setting something 🤔